Sources/DataStack+Migration.swift

```diff
@@ -806,11 +811,23 @@ public extension DataStack {
                 resultingItemURL: nil
             )
             
+            if fileManager.fileExists(atPath: temporaryExtenralStorageURL.path) {
+                let extenralStorageURL = fileURL.deletingLastPathComponent().appendingPathComponent(externalStorageDirName,  isDirectory: true)
+                try fileManager.replaceItem(
```

Comment:
Moving `temporaryFileURL`  and `temporaryExternalStorageURL` should probably be made an atomic operation somehow. If the external storage fails we cannot rollback the main DB. In the mean time, this should be OK in most cases.